RubberduckCodeAnalysis/Resources.Designer.cs

```diff
@@ -64,16 +63,43 @@ internal class Resources {
         /// <summary>
         ///   Looks up a localized string similar to COM Management.
         /// </summary>
-        internal static string AnalyzerCategory {
+        public static string AnalyzerCategory {
             get {
                 return ResourceManager.GetString("AnalyzerCategory", resourceCulture);
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Chained Wrappers.
+        /// </summary>
+        public static string ChainedWrapperDescription {
+            get {
+                return ResourceManager.GetString("ChainedWrapperDescription", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Chained Wrappers.
+        /// </summary>
+        public static string ChainedWrapperMessageFormat {
+            get {
+                return ResourceManager.GetString("ChainedWrapperMessageFormat", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Chained Wrappers.
+        /// </summary>
+        public static string ChainedWrapperTitle {
+            get {
+                return ResourceManager.GetString("ChainedWrapperTitle", resourceCulture);
+            }
+        }
+        
         /// <summary>
         ///   Looks up a localized string similar to COM-visible classes must have an explicit ClassInterface attribute and be set to `None`. This is required to avoid verisoning problems..
         /// </summary>
-        internal static string MissingClassInterfaceDescription {
+        public static string MissingClassInterfaceDescription {
```

Comment:
Why did the accessability change here and at a lot of other places?

Reply:
TBH, not sure. I'll try changing that back (I set the accessibility in the resx file).